mautic/mautic
# Fix UTM content longer like 191 characters

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Our customer add utm content longer like 191 characters, and then click from emails return 

`SQLSTATE[22001]: String data, right truncated: 1406 Data too long for column 'utm_content' at row 1`




#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Add utm tags to email
3. Add utm_content tag longer liek 191 characters
4. Send email
5. Click on the link from email
6. See like redirect link does not work
7. After PR should work




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11317"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
